test/unit/commandsAmqp-test.js

```diff
@@ -182,7 +182,7 @@ describe('AMQP Transport binding: commands', function() {
             setTimeout(function() {
                 contextBrokerMock.done();
                 done();
-            }, 200);
+            }, 1000);
```

Comment:
Has this timeout being increated to reduce the probability of fail in the "glitching AMQP" test at travis CI?

Reply:
Right, I think that maybe the glitching problem was caused because of a very short timeout. This already happened in some unit tests of the LWM2M agent. Let's see if this increase helps to reduce the frequency of this issue and if not we can revert it easily.